scripts/extensions/videoEditor/src/interfaces.ts

```diff
@@ -0,0 +1,28 @@
+import {Crop} from 'react-image-crop';
+
+export interface ICrop extends Crop {
+    scale?: number;
```

Comment:
It use for calculate the drop size when we have do both action drop and rotate. When we rotate the video, it will scale to fit with window video-editor. That is the scale value of video (ICrop.scale), we store and use it for calculate the drop size after rotated.